.gitignore

```diff
@@ -22,3 +22,4 @@ debug
 /win_install.log
 /linux_install.log
 /src/default_excludes.mk
+.vscode
```

Comment:
Don;t know how... without full rewrite histore of commits and lost this conversation.